draw-steel.mjs

```diff
@@ -80,6 +80,10 @@ Hooks.once("init", function () {
     makeDefault: true,
     label: "DRAW_STEEL.Sheet.Labels.ActiveEffect",
   });
+  foundry.applications.apps.DocumentSheetConfig.registerSheet(CombatantGroup, DS_CONST.systemID, applications.sheets.DrawSteelCombatantGroupConfig, {
```

Comment:
Let's just destructure `DocumentSheetConfig` at the top here and rework this entire giant block of sheet registering/unregistering to make it more readable and keep the line lengths down.